package.json

```diff
@@ -119,7 +119,8 @@
     "sharness-full": "make -sC ./sharness prove PROVE_OPTS=-vf TEST_OPTS='-v --chain-lint --long'",
     "coverage": "yarn run unit --coverage",
     "flow": "flow",
-    "lint": "eslint src config --max-warnings 0"
+    "lint": "eslint src config --max-warnings 0",
+    "prepublishOnly": "yarn build"
```

Comment:
Is this a yarn feature? That it runs this before publish?

Reply:
Yep!